src/test/java/com/faforever/integration/factories/PlayerFactory.java

```diff
@@ -3,14 +3,26 @@
 import com.faforever.api.data.domain.Player;
 import com.faforever.api.data.domain.User;
 import com.faforever.integration.TestDatabase;
+import lombok.Builder;
+import org.springframework.util.Assert;
+
+import static org.junit.Assert.assertEquals;
 
 public class PlayerFactory {
-  public static Player createPlayer(String login, TestDatabase database) throws Exception {
+  private static final String DEFAULT_USERNAME = "JUnitTestUser_PlayerFactory";
+  private static final String DEFAULT_PASSWORD = "testPassword";
+
+
+  @Builder
+  private static Player create(String login, String password, TestDatabase database) throws Exception {
```

Comment:
Codacy has problems with the `@Builder` from lombok ...
![image](https://cloud.githubusercontent.com/assets/1217999/26024381/649deb1c-37d0-11e7-8c56-ac43ecc468ac.png)